### ISSUE TITLE
[crash] Something went wrong

### ISSUE THREAD

**Issue by HarshS1611** [kubeshop/monokle#1843]
**Describe the bug**

**Steps to reproduce**

**Stacktrace** 

 ```
Cannot set properties of undefined (setting 'cluster')
The error may be correlated with this previous error:
TypeError: Cannot set properties of undefined (setting 'cluster')
    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:331671:21
    at Array.forEach (<anonymous>)
    at Function.unflatten (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:331640:23)
    at mergeConfigs (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:59051:237)
    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:58666:4928
    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644675:25
    at memoized (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644689:40
    at Object.memoized [as current] (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at Object.checkForUpdates [as onStateChange] (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:634742:48)


 TypeError: Cannot set properties of undefined (setting 'cluster')
The error may be correlated with this previous error:
TypeError: Cannot set properties of undefined (setting 'cluster')
    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:331671:21
    at Array.forEach (<anonymous>)
    at Function.unflatten (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:331640:23)
    at mergeConfigs (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:59051:237)
    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:58666:4928
    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644675:25
    at memoized (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644689:40
    at Object.memoized [as current] (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at Object.checkForUpdates [as onStateChange] (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:634742:48)


    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:331671:21
    at Array.forEach (<anonymous>)
    at Function.unflatten (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:331640:23)
    at mergeConfigs (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:59051:237)
    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:58666:4928
    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644675:25
    at memoized (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644689:40
    at memoized (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at useSelectorWithStoreAndSubscription (file:///C:/Program%20Files/Kubeshop/Monokle/resources/app.asar/build/bundle.main.js:634709:30)
```

**Comment by topliceanurazvan** [comment]
Hello @HarshS1611 , thank you for reporting this crash!
If possible, could you please help us with some more details of how this happened and how it can be reproduced?

**Comment by HarshS1611** [comment]
The application got crashed when I was providing a new path for the minikube cluster kubeconfig file . After I selected the folder after few seconds the application got crashed .

**Comment by topliceanurazvan** [comment]
Thank you @HarshS1611 , we are investigating the problem.

**Comment by devcatalin** [comment]
Hi @HarshS1611, we released the `1.7.5` version of Monokle which includes a fix for this error.
You can download it here: https://github.com/kubeshop/monokle/releases/tag/v1.7.5
I'm closing this issue but please let us know if the problem persists after 1.7.5.
Thank you!
